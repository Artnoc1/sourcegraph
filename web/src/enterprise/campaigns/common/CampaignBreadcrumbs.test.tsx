import React from 'react'
import { shallow } from 'enzyme'
import { CampaignBreadcrumbs } from './CampaignBreadcrumbs'

describe('CampaignBreadcrumbs', () => {
    test('new', () => expect(shallow(<CampaignBreadcrumbs />)).toMatchSnapshot())

    test('existing', () =>
        expect(
            shallow(<CampaignBreadcrumbs campaign={{ name: 'My campaign', url: 'https://example.com' }} />)
        ).toMatchSnapshot())
})
