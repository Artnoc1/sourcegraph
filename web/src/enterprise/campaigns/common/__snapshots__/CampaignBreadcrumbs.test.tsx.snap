// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampaignBreadcrumbs existing 1`] = `
<nav
  aria-label="breadcrumb"
  className=""
>
  <ol
    className="breadcrumb"
  >
    <li
      className="breadcrumb-item"
    >
      <AnchorLink
        to="/campaigns"
      >
        Campaigns
      </AnchorLink>
    </li>
    <li
      className="breadcrumb-item active"
    >
      <AnchorLink
        to="https://example.com"
      >
        My campaign
      </AnchorLink>
    </li>
  </ol>
</nav>
`;

exports[`CampaignBreadcrumbs new 1`] = `
<nav
  aria-label="breadcrumb"
  className=""
>
  <ol
    className="breadcrumb"
  >
    <li
      aria-current="page"
      className="breadcrumb-item active"
    >
      Campaigns
    </li>
  </ol>
</nav>
`;
